maths/extended_gcd.nim

```diff
@@ -0,0 +1,76 @@
+## Extended Euclidean algorithm
+##
+## Provides the implementation of the function extendedGCD*
```

Comment:
I would remove this sentence. It's a truism.

Reply:
Done in 5e538a26bf1d9615fcc7df464c57d46f59dd6216.